Создать интерфейсы и реализации к ним

anton-chernets opened the pull request:
тип кузова
колесная формула
тип двигателя
тип коробки передач
(можно использовать код предыдущего урока)

Создать трейт производящий расчет мощности в зависимости от объема двигателя и типа двигателя

Commit:
Создать интерфейсы и реализации к ним

тип кузова
колесная формула
тип двигателя
тип коробки передач
(можно использовать код предыдущего урока)

Создать трейт производящий расчет мощности в зависимости от объема двигателя и типа двигателя

Commit:
Создать интерфейсы и реализации к ним

тип кузова
колесная формула
тип двигателя
тип коробки передач
(можно использовать код предыдущего урока)

Создать трейт производящий расчет мощности в зависимости от объема двигателя и типа двигателя

ValeriyShnurovoy commented on index.php:
```diff
@@ -368,4 +368,165 @@ public static function getFileNameWithoutFormat(string $str)
+{
+    public function calcEnginePower(string $engineType, int $engineCapacity = null)
+    {
+        //TODO кажется я не правильно задал типы данных, не пойму как считать со стрингой
```
тут тогда у тебя тип получается, и в зависимости от типа уже какие то расчеты

Commit:
Создать интерфейсы и реализации к ним

тип кузова
колесная формула
тип двигателя
тип коробки передач
(можно использовать код предыдущего урока)

Создать трейт производящий расчет мощности в зависимости от объема двигателя и типа двигателя
